api/v1beta1/activemqartemis_types.go

```diff
@@ -574,6 +574,9 @@ type ActiveMQArtemisStatus struct {
 	//we probably use Deployments as operatorHub shows invalid field podStatus
 	//see 3scale https://github.com/3scale/3scale-operator/blob/8abbabd926616b98db0e7e736e68e5ceba90ed9d/apis/apps/v1alpha1/apimanager_types.go#L87
 
+	//+operator-sdk:csv:customresourcedefinitions:type=status,displayName="Deployment Plan Size"
```

Comment:
Are you not adding the OLM Descriptor on purpose?
```suggestion
	//+operator-sdk:csv:customresourcedefinitions:type=status,displayName="Deployment Plan Size",xDescriptors="urn:alm:descriptor:text"
```

Reply:
I was not really sure why I would add text to this field, I think it will be fine with the default representation of an integer, possibly as a string.